examples/example.py

```diff
@@ -33,7 +36,7 @@ def main():
         choice = input("your choice: ")
 
         if choice == 'east' or choice == 'west':
-            tester = True
+            valid_choice_check = True

```

Comment:
Not for this PR (but write it for a future PR). Think if you can abstract this code to a helper / class outside of this file. It seems like this will be something basic to do in the library.